specification/cognitiveservices/data-plane/Language/stable/2022-05-01/conversationalanalysisauthoring.json

```diff
@@ -0,0 +1,3060 @@
+        },
+        "x-ms-examples": {
+          "Successful Get Train Status": {
+            "$ref": "./examples/conversationalanalysisauthoring/SuccessfulGetTrainStatus.json"
+          }
+        }
+      }
+    },
+    "/authoring/analyze-conversations/projects/{projectName}/train/jobs/{jobId}/:cancel": {
+      "post": {
+        "description": "Triggers a cancellation for a running training job.",
+        "operationId": "ConversationalAnalysisAuthoring_CancelTrainingJob",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ProjectNamePathParameter"
+          },
+          {
+            "in": "path",
+            "name": "jobId",
+            "description": "The job ID.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "The job was successfully canceled."
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Cancel Training Job": {
+            "$ref": "./examples/conversationalanalysisauthoring/SuccessfulCancelTrainingJob.json"
+          }
+        }
+      }
+    },
+    "/authoring/analyze-conversations/projects/global/deletion-jobs/{jobId}": {
+      "get": {
+        "description": "Gets the status for a project deletion job.",
+        "operationId": "ConversationalAnalysisAuthoring_GetProjectDeletionStatus",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "jobId",
+            "description": "The job ID.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The project deletion job result.",
+            "schema": {
+              "$ref": "#/definitions/ConversationalAnalysisAuthoringProjectDeletionJobState"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Get Project Deletion Status": {
+            "$ref": "./examples/conversationalanalysisauthoring/SuccessfulGetProjectDeletionStatus.json"
+          }
+        }
+      }
+    },
+    "/authoring/analyze-conversations/projects/global/languages": {
+      "get": {
+        "description": "Lists the supported languages for the given project type.",
```

Comment:
Are these BCP-47 languages? IF so, would be good to specify that as we've been doing in other APIs. If not, still specify what, please.